src/index.ts

```diff
@@ -139,3 +138,110 @@ export class DynamoDBAutoIncrement extends BaseDynamoDBAutoIncrement {
     return { puts, nextCounter }
   }
 }
+
+/**
+ * Update a history table with an auto-incrementing attribute value in DynamoDB
+ *
+ * @example
+ * ```
+ * import { DynamoDB } from '@aws-sdk/client-dynamodb'
+ * import { DynamoDBDocument } from '@aws-sdk/lib-dynamodb'
+ * import { DynamoDBHistoryAutoIncrement } from '@nasa-gcn/dynamodb-autoincrement'
+ *
+ * const client = new DynamoDB({})
+ * const doc = DynamoDBDocument.from(client)
+ *
+ * const autoIncrementHistory = DynamoDBHistoryAutoIncrement({
+ *   doc,
+ *   counterTableName: 'autoincrementHistoryTable',
+ *   counterTableKey: {
+ *     autoincrementItemID: 1
```

Comment:
Pick an example widget ID other than 1 to make it clear that this is _not_ an initial value, but a primary key value:

```suggestion
 *     widgetID: 42
```